README.md

```diff
@@ -5,3 +5,33 @@
 * `npm install`
 * `npm run local` to start the application with inspectable client code.
 * `npm run dev` to start the application with minified IIFE client code.
+
```

Comment:
This is my comment on the RZ thread. I don't think it should be part of the README.